Makefile

```diff
@@ -191,44 +191,51 @@ ko-build-everything: $(ko-targets)
 
 scorecard-ko:
 	KO_DATA_DATE_EPOCH=$(SOURCE_DATE_EPOCH) KO_DOCKER_REPO=${KO_PREFIX}/scorecard LDFLAGS="$(LDFLAGS)" \
-	ko publish -B \
+	KO_CACHE=/tmp/scorecard ko build -B \
```

Comment:
I'd probably just have a single shared `KOCACHE`, maybe `/tmp/ko` that just gets shared across all of these.

Reply:
I've updated them as /tmp/ko, thank you 🙋🏻‍♂️